src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -414,7 +428,21 @@ private void persistGroupToRootComponents(DeploymentDocument deploymentDocument)
                 groupTopics.remove();
             }
         });
+
+        Topics groupLastDeploymentTopics = config.lookupTopics(GROUP_TO_LAST_DEPLOYMENT_TOPICS);
+        groupLastDeploymentTopics.forEach(node -> {
+            Topics groupTopics = (Topics) node;
```

Comment:
don't blindly cast, check using instanceof first

Reply:
I am using the same pattern as line 423.

Reply:
That doesn't mean that it is good :P

Definitely please do check for the type before casting. This protects us in case we change the format, or if the user breaks our config somehow.